openmrs-19/src/main/java/org/motechproject/openmrs19/resource/impl/EncounterResourceImpl.java

```diff
@@ -80,4 +92,16 @@ private Gson buildGsonWithAdapters() {
                 .registerTypeAdapter(Observation.class, new Observation.ObservationSerializer())
                 .create();
     }
+
+    /**
+     * Rewrites provider from the list to single provider object in encounter. Since OpenMRS version 1.10
+     * providers in encounter are stored on the list.
+     *
+     * @param encounter
+     */
+    private void getProviderFromEncounterProviderList(Encounter encounter) {
```

Comment:
rename this method to setProviderFromEncounterProviderList. Usually get methods should return a value, here you are setting a provider value in encounter object.